lib/ts/string_view.h

```diff
@@ -1220,7 +1220,13 @@ template <class _Type, class _Traits> struct hash<ts::basic_string_view<_Type, _
   size_t
   operator()(string_type const &x) const
   {
-    return hash<typename string_type::const_pointer>()(x.data());
+// not what I would normally do.. but better than
+// making a custom has at the moment
```

Comment:
s/has/hash ?

Reply:
@bryancall What is the question/concern?